packages/css-dev-tools/postcssPluginConfig.js

```diff
@@ -1,3 +1,5 @@
+// require('dotenv').config()
```

Comment:
You may keep this. It allow user to add env variables within a `.env` file, which is a good thing

Reply:
fixed